resources/session03/wsgi/calculator.py

```diff
@@ -0,0 +1,79 @@
+
+
+def num(s):
+    # return some sort of number from a string
+    try:
+        return int(s)
+    except ValueError:
+        return float(s)
+
+def multiply(arg1, arg2):
+    return("{} * {} = {}".format(arg1, arg2, arg1*arg2))
+
+
+def add(arg1, arg2):
+    return("{} + {} = {}".format(arg1, arg2, arg1+arg2))
+
+
+def subtract(arg1, arg2):
+    return("{} - {} = {}".format(arg1, arg2, arg1-arg2))
+
+
+def divide(arg1, arg2):
+    return("{} / {} = {}".format(arg1, arg2, arg1/arg2))
```

Comment:
These four functions feel a bit repetitive to me.  The only real difference between them is the name of the operation being performed (`+`, `-`, `*`, or `/`) and the operator used.  Could you think of a way to reduce the four functions to just one?  Look at the `operator` dict you declare below this and think about what you might do instead.  If you knew about the [operator module](https://docs.python.org/3.5/library/operator.html) from the Python standard library, would it make a difference in your approach?